docs/content/stable/tutorials/build-and-learn/chapter4-going-global.md

```diff
@@ -0,0 +1,350 @@
+menu:
+  stable:
+    identifier: chapter4-going-global
+    parent: tutorials-build-and-learn
+    weight: 5
+type: docs
+---
+
+>**YugaPlus - Going Geo-Distributed**
+>
+>The popularity of the YugaPlus streaming platform has continued to soar. In the United States alone, millions of users watch their favorite movies, series, and sports events daily. With this level of growth, the YugaPlus team faced a new challenge.
+>
+>Many customers complained about the YugaPlus app being unresponsive and slow, especially during peak hours. It turned out that most of these complaints originated from users living on the US West Coast. Their requests had to travel across the country to the US East Coast, which hosted the preferred region of the multi-region YugabyteDB cluster. The high latency between the West and East regions explained the poor user experience.
+>
+>Eventually, the YugaPlus team decided to ensure all customers, regardless of their location, received the same level of experience. They achieved low-latency reads and writes using one of the design patterns for global applications...
+
+In this chapter, you'll learn how to do the following:
+
+* Use latency-optimized geo-partitioning for low-latency reads and writes across all user locations.
+
+**Prerequisites**
+
+You need to complete [chapter 3](../chapter3-tolerating-outages) of the tutorial before proceeding to this one.
+
+## Geo-partition the user library
+
+Using the [latency-optimized geo-partitioning pattern](../../../develop/build-global-apps/latency-optimized-geo-partition/), you can pin user data to cloud regions closest to their physical location. By implementing this strategy, your multi-region application can process read and write requests with low latency across all locations.
+
+The `user_library` table in the YugaPlus movies recommendation service is an excellent candidate for geo-partitioning. The current structure of the table is as follows:
+
+```shell
+docker exec -it yugabytedb-node1 bin/ysqlsh -h yugabytedb-node1 \
+    -c '\d user_library'
+```
+
+```output
+                                Table "public.user_library"
+      Column      |            Type             | Collation | Nullable |      Default
+------------------+-----------------------------+-----------+----------+-------------------
+ user_id          | uuid                        |           | not null |
+ movie_id         | integer                     |           | not null |
+ start_watch_time | integer                     |           | not null | 0
+ added_time       | timestamp without time zone |           | not null | CURRENT_TIMESTAMP
+```
+
+By geo-partitioning this table, you can ensure that its data is distributed across cloud regions closest to the users. Consequently, when a user checks or updates their library with the next movies to watch, the requests will be served from the region where the user resides.
+
+Follow these steps to create a geo-partitioned version of the `user_library` table:
+
```

Comment:
Should we stop the container here instead of doing it later where we restart the app?
1. Use `Ctrl+C` or `docker-compose stop` to stop the YugaPlus application containers.